graph/resolver/discussion.resolvers.go

```diff
@@ -34,6 +35,22 @@ func (r *discussionResolver) Posts(ctx context.Context, obj *model.Discussion) (
 	return posts, nil
 }
 
+func (r *discussionResolver) PostsConnection(ctx context.Context, obj *model.Discussion, after *string) (*model.PostsConnection, error) {
+	/* Hardcode the number of posts per page. This can be changed to be settable by the client in the query. */
+	limit := 50
+
+	/* Sanity check. If no "after" parameter is specified, we set it to a time far into the future, for which
+	   no post can yet have been created (not even cosidering large clocks drift) */
+	if after == nil {
+		futureTime := time.Now().AddDate(1, 0, 0).Format(time.RFC3339)
```

Comment:
Time formatting follow the RFC3339 ISO standard, with is compatible with the timestamp format of PostgreSQL. Sanity checks on the on the input are required as bad client requests can potentially break pages consistency.